src/pages/add-edit/add-edit-page.jsx

```diff
@@ -276,8 +276,9 @@ export default class AddEditEventScene extends React.Component {
         };
         requestMethod(url, newEvent)
           .then(
-            response => this.props.eventSavedSuccessfully(idInfo),
```

Comment:
Rename unused parameters to `_.*` pattern, and add this pattern to `.eslintrc.yml`, to fix “'response' is defined but never used” etc. (`no-unused-vars`) lint errors